app/src/renderer/components/transactions/TmLiStakeTransaction.vue

```diff
@@ -153,7 +149,7 @@ export default {
       const validator = this.validators.find(
         val => val.operator_address === validatorAddr
       )
-      return this.atoms(calculateTokens(validator, shares).toNumber())
+      return pretty(this.atoms(calculateTokens(validator, shares).toNumber()))
```

Comment:
won't this always display `<int>.00` instead of the all decimals ? We want to show the full amount on txs

Reply:
we need to resolve the tx amount on all of them so maybe we can address this on another issue